### PR TITLE
feat: Increase storage

### PR DESCRIPTION
**Pull request by Bhuvanasundari**
How do I increase storage on Chandler beyond 50MB?

### CODE REVIEW

**Comment by Bhuvanasundari** [comment]
I increase the 50MB to 64MB changes occur in storage

File: app/Domains/Settings/ManageStorage/Web/ViewHelpers/StorageIndexViewHelper.php [hunk 1]
```diff
@@ -38,7 +38,7 @@ public static function data(Account $account): array
             ->where('type', File::TYPE_AVATAR)
             ->count();
 
-        $accountLimit = $account->storage_limit_in_mb * 1024 * 1024;
+        $accountLimit = 64 * 1024 * 1024;
```
Comment (asbiin) [inline app/Domains/Settings/ManageStorage/Web/ViewHelpers/StorageIndexViewHelper.php:41]: We don't want to change this, the accountLimit comes from this calculation

File: app/Helpers/StorageHelper.php [hunk 2]
```diff
@@ -25,7 +25,7 @@ public static function canUploadFile(Account $account): bool
 
         $totalSizeInBytes = File::whereIn('vault_id', $vaultIds)->sum('size');
 
-        $accountLimit = $account->storage_limit_in_mb * 1024 * 1024;
+        $accountLimit = 64 * 1024 * 1024;
```
Comment (asbiin) [inline app/Helpers/StorageHelper.php:28]: We don't want to change this, the accountLimit comes from this calculation

**Review by asbiin — CHANGES_REQUESTED** [review]
Thank you, but I think this needs more work.

**Comment by djaiss** [comment]
This is not something that we want.

If you need more storage, simply increase your ENV variable in your `.env` file.

The content of your env file should be 

```
DEFAULT_STORAGE_LIMIT=64
```

**Pull request closed without merging**